### ISSUE TITLE
Failed to download Song

### ISSUE THREAD

**Issue by batuhan-ince** [AsHfIEXE/Spotify-Music-Downloader#4]
Hi there,

I was using this code for downloading songs but suddenly now I can not download and gives me an error.

For example when I try to use it on cmd I type "spotify-web-downloader https://open.spotify.com/track/59WN2psjkt1tyaxjspN8fp" as always, I'm having this issue: [ERROR    22:51:57] (Track 1/1 from URL 1/1) Failed to download "Killing In The Name"

I updated my cookies and ffmpeg but the error still continues. Can you help me with that?

**Comment by AsHfIEXE** [comment]
Please be paitent and stay with me. I'm trying to fix this issue. Can you give me a screenshot?

**Comment by AsHfIEXE** [comment]
Run the command again but with --print-exceptions at the end.

**Comment by vofen430** [comment]
Hi AsHfIEXE,

I hope you're doing well. I recently encountered an issue with the Spotify Web Downloader after reinstalling my operating system. The tool was working fine until a few days ago, but now I’m getting the following traceback error:
```
Traceback (most recent call last):
  File "D:\Python\Python312\Lib\site-packages\spotify_web_downloader\cli.py", line 490, in main
    decryption_key = downloader_song.get_decryption_key(pssh)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Python\Python312\Lib\site-packages\spotify_web_downloader\downloader_song.py", line 71, in get_decryption_key
    license = self.downloader.spotify_api.get_widevine_license_music(challenge)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Python\Python312\Lib\site-packages\spotify_web_downloader\spotify_api.py", line 112, in get_widevine_license_music
    check_response(response)
  File "D:\Python\Python312\Lib\site-packages\spotify_web_downloader\utils.py", line 10, in check_response
    _raise_response_exception(response)
  File "D:\Python\Python312\Lib\site-packages\spotify_web_downloader\utils.py", line 14, in _raise_response_exception
    raise Exception(
Exception: Request failed with status code 403:
[INFO     01:51:54] Done (1 error(s))
```
This error started occurring only after I reinstalled my system. The downloader worked perfectly two days ago, before the reinstallation. I have confirmed that cookie.txt was correctly obtained and placed in the ./ directory as per the usual process. Could this be related to missing dependencies, or possibly a change in API authentication that I might need to update? Do you have any suggestions on how I could resolve this issue?

Thank you for your help!

**Comment by AsHfIEXE** [comment]
duplicate of issue #5
